config/dcm2bids.json

```diff
@@ -153,6 +153,22 @@
                 "ImageType": ["*", "*", "P", "*", "*"],
                 "EchoNumber": "6"
             }
+        },
+        {
+            "dataType": "rfmap",
+            "modalityLabel": "rfmap_mag",
+            "criteria": {
+                "SeriesDescription": "standard",
+                "ImageComments": "flip_angle_map*"
+            }
+        },
+        {
+            "dataType": "rfmap",
+            "modalityLabel": "rfmap_phase",
+            "criteria": {
+                "SeriesDescription": "standard",
+                "ImageComments": "*phase*"
```

Comment:
that seems too inclusive, but i agree until we have other data to play with (after imposing conventions on the console), that is probably our only option to filter via dcm2bids. 

alternatively, you could do the filtering _after_ the conversion https://github.com/shimming-toolbox/shimming-toolbox-py/issues/58#issuecomment-670171854. I would lean towards the latter approach, given that it provides more flexibility. E.g., what if a researcher uses an RF map sequence with `"SeriesDescription": "myAwesomeRFsequence"`? the current approach would require updating this json file *and* assume the mag/phase data are half-split.

Reply:
I like the idea of separating them after the conversion but will I be able to modify the metadata of the 2 `.nii.gz` files to specify which one is the magnitude and which one is the phase ? Or is it only possible to specify it in the `json` sidecar ?

Reply:
> will I be able to modify the metadata of the 2 .nii.gz files to specify which one is the magnitude and which one is the phase

with your approach (splitting the dicoms and then convert), are the metadata of the niftis different? what field in the nifti metadata are you referring to?

Reply:
I'm referring to the field that correspond to "ImageComments". For example, when I load my `NIfTI` files with `nibabel.load()`, I get a `object_name.header['aux_file']` field where the content is different for the phase ('relative_phase_map') and magnitude ('flip_angle_map_TraRefAm'). That's because `dcm2niix` seems to take the metadata of the first dicom file in the folder and applies it to the final nifti so when running it on a single folder everything is merged under one type of metadata.

Reply:
> I'm referring to the field that correspond to "ImageComments". For example, when I load my NIfTI files with nibabel.load(), I get a object_name.header['aux_file'] field where the content is different for the phase ('relative_phase_map') and magnitude ('flip_angle_map_TraRefAm'). That's because dcm2niix seems to take the metadata of the first dicom file in the folder and applies it to the final nifti so when running it on a single folder everything is merged under one type of metadata.

i see. that's a problem of the "convert everything to nifti and then split" approach. However, given that it only concerns isolated cases and that this will (hopefully) be solved by proper dicom output from the scanner, maybe we should still go ahead with the splitting of nifti _after_ the dicom conversion, and edition of the json file to update the fields that will be useful for RF shimming down the line. What do you think?

Reply:
Yes as you said we might probably find a way to get proper dicoms on the scanner so we can go with that solution for now.